deepmatcher/data/iterator.py

```diff
@@ -20,10 +20,14 @@ def __init__(self,
                  **kwargs):
         if sort_in_buckets is None:
             sort_in_buckets = train
+        if kwargs.pop('device', None) == 'cpu':
+            device = -1
```

Comment:
Isn't int values for device deprecated? https://github.com/pytorch/text/blob/f34e4fbad1b40627dfdc92c6eaf56969cba77c06/torchtext/data/iterator.py#L66